docs/eip/5559.md

```diff
@@ -0,0 +1,447 @@
+---
+ERC: "5559"
```

Comment:
Please follow the format, including capitalization and quotes, from the original: https://raw.githubusercontent.com/ethereum/ercs/master/ERCS/erc-5559.md